.gitlab-ci.yml

```diff
@@ -23,5 +28,7 @@ test_app:
   script:
     - apt-get update && apt-get install -y git curl libmcrypt-dev default-mysql-client
     - pip install -r requirements.txt
-    - cd thr && pytest
+    - echo "Sleeping for 15 seconds.."; sleep 15;
```

Comment:
This makes Kieron sad, but you knew that

Reply:
I didn't ignore your suggestions :)
I spent some time trying to mock ES using ElasticMock, it didn't go well, I had connections and mapping issues which made running test quite complicated, then I found this article: https://yanglinzhao.com/posts/test-elasticsearch-in-django/ It's the only clean article I found dealing with Django + elasticsearch tests, the author claims that, I quote:

> mocking won't be that useful anyway because this type of tests usually fails to capture the subtle nuances in behaviour of the system under tests and The types of changes we'd often make to these functions involve tweaking the Elasticsearch DSL query JSON, changing global Elasticsearch config, modifying field mapping, or upgrading the Elasticsearch version. In all these cases, mock based tests don't really offer any guard against bugs, because they (by design) bypass all Elasticsearch related code execution paths.